packages/shared/src/components/drawers/Drawer.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactElement, ReactNode, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { RootPortal } from '../tooltips/Portal';
+import useDebounce from '../../hooks/useDebounce';
+
+export enum DrawerPosition {
+  Bottom = 'bottom',
+  Top = 'top',
+}
+
+interface ClassName {
+  overlay?: string;
+  drawer?: string;
+}
+
+export interface DrawerProps {
+  children: ReactNode;
+  className?: ClassName;
+  position?: DrawerPosition;
+  closeOnOutsideClick?: boolean;
```

Comment:
From the storybook when this is true it also closes if you click on the items, not sure this is intended behaviour?

Reply:
Oh, is it. That should not be the case. Let me check quickly.

Reply:
Should be fixed now 🚀

Reply:
Yes validated! Thanks :D
Really nice to see Storybook being so easy to test now :D

Reply:
True, we are now able to say confidently that things are working 🚀